### DISCUSSION TITLE
clicking new note green plus button in app should automatically move curser to new note body

### DISCUSSION THREAD

**Discussion by tweaksalot** [standardnotes/forum#3037] [General]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I click the + new note green button in the app that I have to also click the window on the right to start typing the new note. A great efficiency habit I picked up from nvalt was when you clicked new note, it automatically moved the insertion cursor to the note so you can start typing right away. Right now in SN, it takes two clicks to start typing a new note.

**Describe the solution you'd like**
Click green new note button, move insertion cursor to new note body.

**Describe alternatives you've considered**
The two steps mentioned above

**Additional context**
I wonder if it's a bug or something else is interrupting the new note flow being able to type in the body once you clicked new note since one can notice that immediately after clicking new note, the cursor very very very briefly shows up in the new note body then goes away - however starting to type does nothing unless you click in the note body.
Mac build 3.150.54

**Comment by effieeee** [comment]
Hi @ulte , I'm afraid I can't test on Mac at the moment, but may I know if you experience this when you use the web app as well? (app.standardnotes.com)

I'm afraid I couldn't reproduce this so far on the latest desktop app (v3.153.12), though this is on Windows. I'm also concerned that your new note button appears green, when it should be blue on the version you mentioned here. Are you using a custom theme on your end?
